console/command.go

```diff
@@ -397,24 +399,31 @@ var getAddressesByAccountCmd = &cobra.Command{
 	},
 }
 var importPriKeyCmd = &cobra.Command{
-	Use:   "importprivkey {priKey} {pripassword}",
+	Use:   "importprivkey {priKey} {pripassword} {raw(hex), wif(Base58); Private Key format; default: raw}",
 	Short: "import priKey ",
 	Example: `
-		importprivkey priKey pripassword
+		importprivkey priKey pripassword ef235aacf90d9f4aadd8c92e4b2562e1d9eb97f0df9ba3b508258739cb013db2
+		importprivkey priKey pripassword 5HueCGU8rMjxEXxiPuD5BDku4MkFqeZyd4dZ1jvhTVqvbTLvyTJ wif
```

Comment:
Please use `flag` to handle the format type such as 
`importtprvikey {prikey} --password=xxx --format=wif`
so that 
`importprvikey --password=xxx --format=wif  {prikey}`  or ` importprvikey -P xxx -f wif  {prikey} ` etc... are the same thing.
see https://github.com/spf13/cobra#conceptss  for details

Reply:
00d9101886cdd59bcfd6828219e6407af608c6b9

Reply:
BTW, I didn't make password as flag since I think it is a required argument